### PR TITLE
Misc css fixes

### LINKED ISSUE #1512: Feedback Form
Need a method of receiving feedback about the manager.

### LINKED ISSUE #1546: Move users/billing/settings out of profile dropdown
Discuss:

Separate Icon or Primary Nav Account

### PR DESCRIPTION
**Pull request by eatonphil**
Closes #1546, closes #1512, closes #1741

* Shrink notification badge number size
* Drop Linode Manager text from header
* Add named sections to session menu
* Add feedback link to session menu
* Add hover and make session menu items easier to click
* Adjust card headers and page background color slightly so card headers stand out against background and against buttons inside of them
* Fix distro dropdowns
* Drop glish from linode dashboard page for now
* Status dropdown uses dropdown component
* Domains table capitalizes domain type

screen shot 2017-05-11 at 12 01 59 am

### CODE REVIEW

File: src/layouts/OAuth.js [hunk 1]
```diff
@@ -58,7 +58,6 @@ export class OAuthCallbackPage extends Component {
       });
       const json = await resp.json();
       this.setSession(json.access_token, json.scopes, username, email, timezone);
-      await dispatch(account.one());
```
Comment (eatonphil) [inline src/layouts/OAuth.js]: I need to add account.one() to preload to the account settings page now.

File: components/lists/controls/MassEditControl.js [hunk 2]
```diff
@@ -46,10 +46,10 @@ export default class MassEditControl extends Component {
         checked={allSelected}
         disabled={noneSelected}
         options={massEditOptions.map((option) => {
-          return {
+          return option && {
```
Comment (mparke) [inline components/lists/controls/MassEditControl.js]: does this return `undefined` options into the list? if so `filter` out options before hand?

File: src/components/SessionMenu.js [hunk 3]
```diff
@@ -6,33 +6,40 @@ export default function SessionMenu(props) {
   return (
     <div className={`SessionMenu ${open ? 'SessionMenu--open' : ''}`}>
       <ul className="SessionMenu-body">
-        <li className="list-unstyled SessionMenu-menu-item">
+        <li className="SessionMenu-header">My User</li>
```
Comment (eatonphil) [inline src/components/SessionMenu.js]: Drop this header.

File: src/components/SessionMenu.js [hunk 4]
```diff
@@ -6,33 +6,40 @@ export default function SessionMenu(props) {
...
+        <li className="SessionMenu-item">
+          <Link to="/support">Support</Link>
         </li>
-        <li className="list-unstyled SessionMenu-menu-item">
-          <Link to="/billing">Billing</Link>
+        <li className="SessionMenu-item">
+          <a href="mailto:feedback@linode.com">Submit Feedback</a>
+        </li>
+        <hr />
+        <li className="SessionMenu-header">My Account</li>
```
Comment (eatonphil) [inline src/components/SessionMenu.js]: s/My /g

File: src/components/SessionMenu.js [hunk 5]
```diff
@@ -6,33 +6,40 @@ export default function SessionMenu(props) {
...
+          <Link to="/billing">Billing</Link>
         </li>
-        <li className="list-unstyled SessionMenu-menu-item">
-          <Link to="/support">Support</Link>
+        <li className="SessionMenu-item">
+          <Link to="/users">Users</Link>
         </li>
         <hr />
-        <li className="list-unstyled SessionMenu-menu-item">
+        <li className="SessionMenu-header">Information</li>
```
Comment (eatonphil) [inline src/components/SessionMenu.js]: Resources

File: src/layouts/OAuth.js (hunk 1 above)
Comment (eatonphil) [inline src/layouts/OAuth.js]: Already done apparently.

**Comment by na3d** [comment]
Padding-top fix for the section headers then good to merge.

**Comment by eatonphil** [comment]
@na3d you know my conditions.

**Comment by na3d** [comment]
Yeah, more of a design thing, so not critical, we can easily change later.

**Comment by eatonphil** [comment]
Thanks!

**Pull request merged**